src/odemis/acq/stream/_static.py

```diff
@@ -46,45 +46,104 @@ class StaticStream(Stream):
     def __init__(self, name, raw):
         """
         Note: parameters are different from the base class.
-        image (DataArray): static raw data.
-        raw (None or list of DataArrays): raw data to be used at initialisation
-          by default, it will contain no data.(None or list of DataArrays)
-        """
+        raw (DataArray, DataArrayShadow or list of DataArray): The data to display.
+        """
+        if isinstance(raw, model.DataArray):
+            raw = [raw]
+        elif isinstance(raw, list):
+            if len(raw) > 0 and not isinstance(raw[0], model.DataArray):
+                raise ValueError("'raw' must be a list of DataArray")
+        elif isinstance(raw, model.DataArrayShadow):
+            if hasattr(raw, 'maxzoom'):
+                md = raw.metadata
+                # get the pixel size of the full image
+                ps = md[model.MD_PIXEL_SIZE]
+                default_mpp = ps[0] * raw.maxzoom ** 2
+                # sets the mpp as the X axis of the pixel size of the full image
+                self.mpp = model.FloatVA(default_mpp, setter=self._set_mpp)
```

Comment:
Change .mpp to a FloatContinous. The range should be from ps[0] to default_mpp (=max_mpp).